app/views/assignments/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <% content_for :title, t('assignments.manage_course_work') %>
 
-<% if @current_user.admin? %>
+<% if @current_user.admin? || @current_user.ta? %>
```

Comment:
Shouldn't this be conditional on whether they have permission to edit assignments or not (not just if they are a grader or not)?